spec/migrations/20200512201614_update_chargeback_startpage_spec.rb

```diff
@@ -0,0 +1,49 @@
+require_migration
+
+describe UpdateChargebackStartpage do
+  let(:feature_stub) { migration_stub :MiqProductFeature }
+  let(:user_stub) { migration_stub :User }
+
+  migration_context :up do
+    describe 'starting page replace' do
+      it 'replaces user starting page if chargeback/explorer' do
+        feature_stub.create!(:identifier => 'chargeback')
```

Comment:
Why are we creating the feature if we never use it?

Reply:
@Fryguy addressed your comments